engine/consensus/approvals/testutil.go

```diff
@@ -72,9 +72,8 @@ func (s *BaseApprovalsTestSuite) SetupTest() {
 	s.IncorporatedBlock = unittest.BlockHeaderWithParentFixture(s.Block)
 
 	// compose incorporated result
-	s.IncorporatedResult = unittest.IncorporatedResult.Fixture(
-		unittest.IncorporatedResult.WithResult(result),
-		unittest.IncorporatedResult.WithIncorporatedBlockID(s.IncorporatedBlock.ID()))
+	incorporatedResult := flow.NewIncorporatedResult(s.IncorporatedBlock.ID(), result)
+	s.IncorporatedResult = incorporatedResult
```

Comment:
@jordanschalm am I missing smth or why lint is failing here `with write to IncorporatedResult field outside constructor` if nothing being changed after using constructor function?

Reply:
You're not missing anything -- this looks like a bug in the linter. It is using the field type to check for whether it is protected, for some reason, rather than the outer struct type. I created an issue: https://github.com/onflow/flow-go/issues/7376. In the meantime you can ignore this line

```suggestion
	s.IncorporatedResult = incorporatedResult // nolint:structwrite TODO(7376): linter bug here - remove this after fixing bug
```

Reply:
@AndriiDiachuk This issue should be fixed by https://github.com/onflow/flow-go/pull/7386

Reply:
thanks for the quick fix.